libs/utils/utils.js

```diff
@@ -419,6 +419,40 @@ export function decorateSVG(a) {
     return a;
   }
 }
+function hasUrlWithPipe(str) {
+  const regex = /^https?:\/\/.*\|.*/;
+  return regex.test(str);
+}
+
+export function decorateImageLinks(a) {
+  const images = a.querySelectorAll('img');
+  if (!images.length) return a;
+  [...images].forEach((img) => {
+    // Check if the image has an alt tag with a url & pipe - 'https... |'
+    if (!(hasUrlWithPipe(img.alt))) return a;
+    try {
+      // //Mine for URL and alt text
+      const splitText = img.alt.split('|');
+      const textUrl = new URL(splitText.shift().trim());
+      const altText = splitText.join('|').trim();
+      if (altText) img.alt = altText;
+
+      const pic = img.closest('picture');
+      const picParent = pic.parentElement;
+      const aTag = createTag('a', { href: textUrl, class: 'desc-link' }, pic);
+      if (aTag.href.includes('#_blank')) {
```

Comment:
```suggestion
      if (aTag.hash === '#_blank') {
```

Reply:
Maybe a utility function is needed for this since the same condition is [used here too](https://github.com/adobecom/milo/blob/27100d99661e0c8237e82be6bbf50849f97a2bb5/libs/utils/utils.js#L505-L508)

Reply:
See my comment lower where you are calling `decorateImageLinks(area)`.

Reply:
@rgclayton  - I refactored this and called the 'decorateImageLinks()' in the `decorateLinks` func and thus inherited the target functionality. Checks out from my testing, I could see moving this to a single util if needed for multiple functions down the road. LMK if you have any thoughts.